<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 161, 54);
	border: 0.1px solid grey;
color: white;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 161, 54);
	border: 0.1px solid grey;
color: white;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: rgb(242, 82, 208);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 161, 54);
	border: 0.1px solid grey;
color: white;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>510</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, y2:1, 
								stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	border: 0.5px solid grey;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, y2:1, 
								stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 161, 54);
	border: 0.1px solid grey;
color: white;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
	background-color: rgb(215, 215, 215);
border: 0.1px solid grey;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
	background-color: rgb(215, 215, 215);
border: 0.1px solid grey;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 13pt &quot;Helvetica&quot;;
	font: 22pt &quot;.AppleSystemUIFont&quot;;
	background-color: rgb(215, 215, 215);
border: 0.1px solid grey;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 161, 54);
	border: 0.1px solid grey;
color: white;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: rgb(219, 110, 208);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>509</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>509</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>509</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: black;
color: rgb(255, 0, 200);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_closed_brace">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: rgb(219, 110, 208);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_opened_brace">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: rgb(219, 110, 208);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scale">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>240</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;	
	background-color: rgb(219, 110, 208);
	border: 0.1px solid grey;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
